internal/infrastructure/price-feeder/domain.go

```diff
@@ -0,0 +1,35 @@
+package pricefeederinfra
```

Comment:
same for this file, i totally understand the naming but to prevent any kind of confusion, we can call it price_feed.go or feed.go, as you wish.